features/counter-reset-reversed.yml

```
@@ -0,0 +1,6 @@
+name: Reversed counter-reset
+description: "The `reversed()` CSS function is used with `counter-reset` to create a counter that counts down."
```

Avoiding "is used to" 

```suggestion
description: "The `reversed()` CSS function for `counter-reset` creates a counter that counts down."
```